docker/Dockerfile.backend

```
@@ -0,0 +1,13 @@
+FROM node:20-alpine
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN npm install --only=production
+
+COPY server ./
+
+EXPOSE 3005
```

Líneas 9-11 (Añadir usuario no-root y health check, reemplazar CMD)  deberías añadir un usuario no-root por seguridad:
  ```suggestion
  RUN addgroup -g 1001 -S nodejs && \
      adduser -S nodejs -u 1001 && \
      chown -R nodejs:nodejs /app

  USER nodejs

  ENV APP_PORT=3005
  EXPOSE 3005

  HEALTHCHECK --interval=30s --timeout=3s --start-period=5s \
    CMD wget --no-verbose --tries=1 --spider http://localhost:3005/health || exit 1

  CMD ["node", "server/index.js"]
  ```